schemas/com.snowplowanalytics.monitoring.batch/load_succeeded/jsonschema/2-0-1

```diff
@@ -0,0 +1,135 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a data load succeeding",
+  "self": {
+    "vendor": "com.snowplowanalytics.monitoring.batch",
+    "name": "load_succeeded",
+    "format": "jsonschema",
+    "version": "2-0-1"
+  },
+
+  "type": "object",
+  "properties": {
+    "shredding": {
+      "description": "Information about the batch taken from transformation step, isomorphic to shredding_complete schema",
+      "type": "object",
+      "properties": {
+        "base": {
+          "description": "Blob storage path to the root of the batch",
+          "type": "string",
+          "format": "uri",
+          "maxLength": 1024
+        },
+        "compression": {
+          "description": "File compression type",
+          "enum": ["GZIP", "NONE"]
+        },
+        "types": {
+          "description": "Set of Iglu URIs and the format they were shredded into",
+          "type": "array",
+          "items": {
+            "properties": {
+              "schemaKey": {
+                "description": "Iglu URI",
+                "type": "string",
+                "maxLength": 256
+              },
+              "format": {
+                "description": "File format in which the entities were shredded to",
+                "enum": ["TSV", "JSON", "WIDEROW", "PARQUET"]
+              },
+              "shredProperty": {
+                "description": "Field types of enriched event that can be shredded (self-describing JSONs)",
+                "enum": ["SELFDESCRIBING_EVENT", "CONTEXTS"]
+              }
```

Comment:
Please can you describe to me a bit more about what this `shredProperty` means?  I don't understand it at the moment, but I might have missed something obvious.

Reply:
It is direct mapping of [ShredProperty in Analytics SDK](https://github.com/snowplow/snowplow-scala-analytics-sdk/blob/master/src/main/scala/com.snowplowanalytics.snowplow.analytics.scalasdk/Data.scala#L45). 

Basically, it specifies the type of self-describing JSON field in the event (context or unstruct_event). I've used `SELFDESCRIBING_EVENT` instead of `UNSTRUCT_EVENT` because atomic schema is also part of the `types` list and we need to set `shredProperty` for it too. Setting `shredProperty` to `UNSTRUCT_EVENT` for atomic schema wasn't seem right therefore I named it `SELFDESCRIBING_EVENT`.  

Since it is named as `shredProperty` in Analytics SDK, I also named it like that in here. But it can be named better in here maybe. Maybe `shredType` ? But using `type` word might create some confusion since we are already in `types` list.

Reply:
This new field is put here to create column names in new Snowflake Loader in same way with old Snowflake Loader. Old Snowflake Loader is using [SnowplowEvent.transformSchema from Analytics SDK](https://github.com/snowplow-incubator/snowplow-snowflake-loader/blob/master/transformer/src/main/scala/com/snowplowanalytics/snowflake/transformer/Transformer.scala#L43) to create new column names and `transformSchema` needs `shredProperty`. In order to create column names in same way with old Snowflake Loader, new  `shredProperty` field is introduced here.

Reply:
Ah, I understand now.  I missed before that `shredProperty` is part of the `types` array; I thought it was a single shredProperty for the entire batch.

I understand your doubts about the name, and I understand it is made more messy because atomic schema is part of types.

Would it be a good idea to remove atomic schema completely from this types array?  I know that makes it a big breaking change, but it might be easier to reason about the semantic meaning of this array.

Reply:
I can't see any downsides for removing atomic schema. 

@chuwy May I ask if you have any objections ?